tests/functional/LocaleTest.php

```diff
@@ -10,46 +10,81 @@
  */
 class LocaleTest extends CommandUnishTestCase
 {
+    /**
+     * File name of Gettext PO source file.
+     *
+     * @var string
+     */
+    protected $sourceFile;
 
-    public function testLocaleImport()
+    public function setUp()
```

Comment:
I just noticed that setUp() is now installing Drupal twice, once for each test method. See https://github.com/drush-ops/drush/blob/50ae39d721fd45727349ab5915453c61925764a4/tests/functional/RsyncTest.php#L21-L23 for a simple way to avoid that (just put your logic inside a `getSites()` check.